### PR TITLE
JSON Schema Migration

### LINKED ISSUE #234: JSON Schema Migration
Provide a tool to perform migration against JSON schema (_e.g._ if users stored form definitions into database, files).

### PR DESCRIPTION
**Pull request by wo0dyn**
## Review

This PR closes #234

* [x] Tests
* [x] Docs/comments
* [x] Migration(s)
* [x] `CHANGELOG.rst` Updated
* [ ] Delete your branch

### CODE REVIEW

File: demo/demo/settings.py [hunk 1]
```diff
@@ -89,7 +89,7 @@
 DATABASES = {
     'default': {
         'ENGINE': engine,
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'NAME': os.path.join('db.sqlite3'),
```
Comment (brunobord) [inline demo/demo/settings.py]: you don't need no join here.

File: tox.ini [hunk 2]
```diff
@@ -1,11 +1,11 @@
 [tox]
 envlist =
-    django18-py{27,34,35}
-    django18-py{27,34,35}-drf33
-    django19-py{27,34,35}
-    django19-py{27,34,35}-drf33
-    django110-py{27,34,35}
-    django110-py{27,34,35}-drf33
+    django18-py{27,35}
```
Comment (brunobord) [inline tox.ini]: I see what you did there... we might have to state this somewhere in the documentation (README and ???) that Python3.4 is not supported anymore.

File: demo/demo/settings.py (hunk 1 above)
Comment (wo0dyn) [inline demo/demo/settings.py]: Yeah. Actually, I'm going to set the right name for the test database. That was the issue that kinda forced me to remove `BASE_DIR`. When I launched unittests, Django was trying to create a file `test_/path/to/db.sqlite3`. I'll amend this.

File: tox.ini (hunk 2 above)
Comment (wo0dyn) [inline tox.ini]: I'll add a warning. Correct.

Comment (wo0dyn) [inline tox.ini]: Done.

**Pull request merged**